graphql_compiler/compiler/emit_match.py

```diff
@@ -198,3 +198,64 @@ def emit_code_from_ir(match_query):
     query_data.appendleft(_construct_output_to_match(match_query.output_block))
 
     return u' '.join(query_data)
+
+
+def emit_code_from_multiple_match_queries(match_queries):
+    """Return a MATCH query string from a list of MatchQuery namedtuples."""
+    optional_variable_base_name = '$optional__'
+    union_variable_name = '$result'
+    query_data = deque([u'SELECT EXPAND(', union_variable_name, u')', u' LET '])
+
+    optional_variables = []
+    sub_queries = [emit_code_from_single_match_query(match_query)
+                   for match_query in match_queries]
+    for (i, sub_query) in enumerate(sub_queries):
+        variable_name = optional_variable_base_name + str(i)
+        variable_assignment = variable_name + u' = ('
+        sub_query_end = u'),'
+        query_data.append(variable_assignment)
+        query_data.append(sub_query)
+        query_data.append(sub_query_end)
+        optional_variables.append(variable_name)
+
+    query_data.append(union_variable_name)
+    query_data.append(u' = UNIONALL(')
+    query_data.append(u', '.join(optional_variables))
+    query_data.append(u')')
+
+    return u' '.join(query_data)
+
+
+def emit_code_from_ir(compound_match_query):
+    """Return a MATCH query string from a CompoundMatchQuery."""
+    # If the compound match query contains only one match query,
+    # just call `emit_code_from_single_match_query`
+    # If there are multiple match queries, construct the query string for each
+    # individual query and combine them as follows.
+    #
+    # SELECT EXPAND($result)
+    # LET
+    #    $optional__0 = (
+    #        <query_string_0>
+    #    ),
+    #    $optional__1 = (
+    #        <query_string_1>
+    #    ),
+    #    $optional__2 = (
+    #        <query_string_2>
+    #    ),
+    #
+    #    . . .
+    #
+    #    $result = UNIONALL($optional__0, $optional__1, . . . )
+
+    match_queries = compound_match_query.match_queries
+    if len(match_queries) == 1:
+        query_string = emit_code_from_single_match_query(match_queries[0])
+    elif len(match_queries) > 1:
+        query_string = emit_code_from_multiple_match_queries(match_queries)
+    else:
+        raise AssertionError(u'Received CompoundMatchQuery with an empty list of MatchQueries'
+                             u': {}'.format(match_queries))
```

Comment:
Move the `: ` part to the previous line, and make this line be `u'{}'.format(match_queries))`